sklearn/kernel_approximation.py

```diff
@@ -641,3 +641,12 @@ def _get_kernel_params(self):
                                  "or precomputed kernel")
 
         return params
+
+    def _more_tags(self):
+        return {
+            '_xfail_checks': {
+                'check_transformer_preserve_dtypes':
+                'dtypes are preserved but not at a close enough precision',
```

Comment:
Do you remember what's the required tolerance for this to pass?

Reply:
it does not pass even with `rtol=1e-1`